### PR TITLE
Footer newsletter

### PR DESCRIPTION
**Pull request by jakataja**

### CODE REVIEW

File: src/scss/footer/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    @include desktop {
+      display: flex;
+      padding-right: 0;
+    }
+
+    .text-input {
+      background: $white;
+      border: 1px solid $basicInputColor;
+      border-radius: 3px;
```
Comment (jacekpolak) [inline src/scss/footer/_footer.scss]: Right now we have $globalBorderRadius in _variables.scss

File: src/scss/footer/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    margin-right: 10px;
+    text-decoration: none;
+
+    @include desktop {
+      vertical-align: middle;
+    }
+
+    .svg-icon {
+      background: $white;
+      border-radius: 3px;
```
Comment (jacekpolak) [inline src/scss/footer/_footer.scss]: here also $globalBorderRadius

File: src/scss/footer/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    display: inline-block;
+    margin-right: 10px;
+    text-decoration: none;
+
+    @include desktop {
+      vertical-align: middle;
+    }
+
+    .svg-icon {
+      background: $white;
```
Comment (jacekpolak) [inline src/scss/footer/_footer.scss]: Add mapping at the top for $white

File: index.html [hunk 4]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
   </div>
 </div>
 
+<footer data-components="Footer" class="footer">
+    <div class="footer__top">
+        <div class="container">
+            <div class="row">
+
+                <div class="footer__newsletter col-sm-12 col-md-6 col-lg-6">
+                    <div class="footer__title">SIGN UP FOR NEWS AND OFFERS</div>
```
Comment (jacekpolak) [inline index.html]: I will switch from UPPERCASE and transform it via CSS

File: index.html [hunk 5]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
...
+                        <button type="submit" class="footer__btn-submit" value="Sumbit email">
+                          <svg class="svg-icon footer__icon--right-arrow">
+                            <use xlink:href="#icon-amway-icons-right-arrow" ></use>
+                          </svg>
+                        </button>
+                    </div>
+                </div>
+
+                <div class="footer__socialmedia col-sm-12 col-md-6 col-lg-6">
+                  <div class="footer__title">CONNECT WITH US ON</div>
```
Comment (jacekpolak) [inline index.html]: UPPERCASE -> CSS transform

File: src/scss/components/footer/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+    @include desktop {
+      display: flex;
+      padding-right: 0;
+    }
+
+    .text-input {
+      background: $white;
+      border: 1px solid $basicInputColor;
+      border-radius: 3px;
```
Comment (jacekpolak) [inline src/scss/components/footer/_footer.scss]: Global variable for border radius now is in _variables.scss

File: src/scss/components/footer/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    margin-right: 10px;
+    text-decoration: none;
+
+    @include desktop {
+      vertical-align: middle;
+    }
+
+    .svg-icon {
+      background: $white;
+      border-radius: 3px;
```
Comment (jacekpolak) [inline src/scss/components/footer/_footer.scss]: Global variable for border radius now is in _variables.scss

File: src/scss/components/footer/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+      display: block;
+      height: 35px;
+      width: 35px;
+    }
+  }
+
+  &__icon {
+    &--email {
+      fill: $alto;
```
Comment (jacekpolak) [inline src/scss/components/footer/_footer.scss]: We can map $alto color into component local variable

File: src/scss/components/footer/_footer.scss [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+    .text-input {
+      background: $white;
+      border: 1px solid $basicInputColor;
+      border-radius: 3px;
+      display: flex;
+      font-size: 1rem;
+      line-height: 2rem;
+      margin-bottom: 0;
+      padding: 7px 10px;
```
Comment (jacekpolak) [inline src/scss/components/footer/_footer.scss]: envelope is not aligned with input and arrow  align-items: center; will help for icon inside "input container" https://snag.gy/DqsYUR.jpg

File: index.html [hunk 10]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
   </div>
 </div>
 
+<footer data-components="Footer" class="footer">
```
Comment (jacekpolak) [inline index.html]: We don't have Footer component in js so let's remove data-component

File: index.html [hunk 11]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
...
+    <div class="footer__top">
+        <div class="container">
+            <div class="row">
+
```
Comment (jacekpolak) [inline index.html]: not necessary space

File: index.html (hunk 4 above)
Comment (jacekpolak) [inline index.html]: I will add here h3 instead of div

File: index.html (hunk 5 above)
Comment (jacekpolak) [inline index.html]: I will add here h3 instead of div

File: index.html [hunk 12]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
...
+                <div class="footer__newsletter col-sm-12 col-md-6 col-lg-6">
+                    <div class="footer__title">SIGN UP FOR NEWS AND OFFERS</div>
+                    <div class="text-input">
+                        <svg class="svg-icon footer__icon--email">
+                          <use xlink:href="#icon-amway-icons-contact-email" ></use>
+                        </svg>
+                        <input class="text-input__focus" type="text" placeholder="Your email address">
```
Comment (yura-matsko) [inline index.html]: add a form tag

File: index.html [hunk 13]
```diff
@@ -256,6 +256,62 @@ <h1 class="h1-title--flood">H1 title flood</h1>
...
+                          <svg class="svg-icon footer__icon--right-arrow">
+                            <use xlink:href="#icon-amway-icons-right-arrow" ></use>
+                          </svg>
+                        </button>
+                    </div>
+                </div>
+
+                <div class="footer__socialmedia col-sm-12 col-md-6 col-lg-6">
+                  <div class="footer__title">CONNECT WITH US ON</div>
+                  <div class="footer__icons">
```
Comment (yura-matsko) [inline index.html]: make it `<ul>`

**Pull request merged**